docs/bundles/SyliusResourceBundle/forms.rst

```diff
@@ -33,6 +33,16 @@ You need to create a simple class:
         }
     }
 
+Create the form's service :
+
+.. code-block:: yaml
+
+    app.book.form.type:
+        class: AppBundle\Form\Type\BookType
+        tags:
+            - { name: form.type }
+        arguments: ['%sylius.model.order.class%']
```

Comment:
@lchrusciel Yes, you were right, i think that it would have crash when you had submitted the form.


I have updated the PR